packages/playwright-core/src/server/chromium/chromium.ts

```diff
@@ -307,10 +307,10 @@ export class Chromium extends BrowserType {
     if (options.devtools)
       chromeArguments.push('--auto-open-devtools-for-tabs');
     if (options.headless) {
-      if (process.env.PLAYWRIGHT_CHROMIUM_USE_HEADLESS_NEW)
-        chromeArguments.push('--headless=new');
-      else
+      if (process.env.PLAYWRIGHT_CHROMIUM_USE_HEADLESS_OLD)
```

Comment:
While passing `--headless=new` to Headless Shell works, and they probably treat it as old, I'm thinking if we should pass `--headless=old` there instead.